### PR TITLE
PayPal in RTP and enhanced payment token

### PR DESCRIPTION
**Pull request by adamaltman**
For corresponding alternative RFC.

### CODE REVIEW

**Review by justnero — COMMENTED** [review]
If multiple merchantIds are available – multiple RTP items with different features would be generated (as in Plaid)

**Comment by adamaltman** [comment]
Here is a sample comment [preview](#)

**Comment by slavcodev-rebilly** [comment]
@Rebilly/js please review

**Review by ashkarpetin — APPROVED** [review]

File: openapi/components/schemas/ReadyToPay/ReadyToPayMethods.yaml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 type: array
 items:
-  oneOf:
+  anyOf:
```
Comment (slavcodev-rebilly) [inline openapi/components/schemas/ReadyToPay/ReadyToPayMethods.yaml:3]: I think this is incorrect, but it was in initial proposal, so I left it as is.

**Review by ghost — APPROVED** [review]

**Pull request merged**